index.js

```diff
@@ -100,65 +141,93 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
-
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+//solution
+function triple(num) {
+  return num * 3;
+}
+console.log(myMap([1,2,3,4,5], triple)) 
+// => [3,6,9,12,15]
 
+console.log(myMap([2,4,6,8], triple))   
+// => [6,12,18,24]
+//
+console.log("%c----------", "color: red") 
 
 // ***** Callbacks - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
+console.log("%cCallbacks - Question 2", "color: red")
 
-// console.log(myMap(["Raffy", "Chase"], greet)) 
-// // => ["Hi, Raffy!", "Hi, Chase!"]
-// console.log("%c----------", "color: red") 
+//solution
+function greet(name) {
+  return `Hi, ${name}`;
+}
+
+console.log(myMap(["Raffy", "Chase"], greet)) 
+// => ["Hi, Raffy!", "Hi, Chase!"]
+//
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Scope & Closures *****
 
-function takeANumber(line, name) {
-  line.push(name)
+// function takeANumber(line, name) {
+//   line.push(name)
+
+//   return `Welcome, ${name}. You are number ${line.length} in line.`
+// }
+
+// function nowServing(line) {
+//   if (!line.length) {
+//     return "There is nobody waiting to be served!"
+//   }
+//   return `Currently serving ${line.shift()}.`
+// }
+
+// const line = []
+
+// ***** Scope & Closures - Question 1 *****
 
-  return `Welcome, ${name}. You are number ${line.length} in line.`
-}
 
+// *** Uncomment the lines below to test
+console.log("%cScope & Closures - Question 1", "color: red")
+
+//solution
+
+const line = [];
+function takeATicketNumber(line) {
+  // let ticketNumber
+  line.push(ticketNumber);
+  return `Welcome, you are ticket number ${line.length}`;
+}
+let ticketNumber;
```

Comment:
You are close here. If we define ticketNumber outside of the function it will be in the global scope and will not reset each time we call the function.